src/connectors/winampify.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+Connector.playerSelector = '#winamp-meta';
+
+Connector.isPlaying = () => Util.hasElementClass('#main-window', 'play');
+Connector.getAlbum = () => Util.getTextFromSelectors('#winamp-meta-album', 'unknown');
```

Comment:
Please use selectors instead of getters.

Reply:
Done C: